public/stylesheets/site/2.0/07-interactions.css

```diff
@@ -21,15 +21,15 @@ fieldset {
   min-width: 0;
 }
 
-/*Guideline: Forms are written by lots of people, and, this is beta, change a LOT. 
-I've written a load of possible different nests to give a reasonably/broadly consistent view whatever goes in, 
+/*Guideline: Forms are written by lots of people, and, this is beta, change a LOT.
+I've written a load of possible different nests to give a reasonably/broadly consistent view whatever goes in,
 but it's probably not complete. */
 
 fieldset, form dl, fieldset dl dl, fieldset fieldset fieldset, fieldset fieldset dl dl, dd.hideme, form blockquote.userstuff {
   display: block;
   background: #ddd;
   border: 2px solid #f3efec;
-  margin: 0.643em;
+  margin: 0.643em auto;
```

Comment:
I'm afraid this isn't a good idea. This doesn't _just_ affect the outermost edges of a form -- it also reduces the left and right margins on all sorts of things _inside_ forms, leading to a cramped or unbalanced appearance, depending on the form. 

For example, here's a prompt meme form with this change:
![margin_auto](https://cloud.githubusercontent.com/assets/907055/22409784/6595af42-e65d-11e6-8e2a-600cb772c73b.png)

And here it is before this change:
![margin_0643](https://cloud.githubusercontent.com/assets/907055/22409787/6e08f684-e65d-11e6-8bc0-507de7d240f2.png)

The new version cuts back on the amount of empty space between the edges of the white portion of the form and the edges of the grey portion. 

In general, I'm not sure the notices and the forms _should_ be the same width. With the notices wider than the forms, it's an additional, subtle reinforcement of the relationship between the two elements, saying "this notice encapsulates everything beneath it."